src/main/menu/show-about.injectable.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { showAbout } from "./menu";
+import showMessagePopupInjectable from "../electron-app/features/show-message-popup.injectable";
+
+const showAboutInjectable = getInjectable({
+  id: "show-about",
+
+  instantiate: (di) =>
```

Comment:
Wouldn't it be easier to read if it was the following:

```ts
instantiate: (di) => showAbout({
  showMessagePopup: di.inject(showMessagePopupInjectable),
}),
```

Reply:
Same as above about code-style. TLDR: we don't do any formatting ourself. It's job for a computer.